docs/security.md

```diff
@@ -15,48 +15,67 @@ Follow these instructions to [authenticate in strict mode](https://docs.mesosphe
 
-    dcos package describe spark --config
+| Variable                         | Description                                     |
+|----------------------------------|-------------------------------------------------|
+| `spark.ssl.enabled`              | Whether to enable SSL (default: `false`).       |
+| `spark.ssl.enabledAlgorithms`    | Allowed cyphers                                 |
+| `spark.ssl.keyPassword`          | The password for the private key                |
+| `spark.ssl.keyStore`             | must be server.jks                              |
+| `spark.ssl.keyStorePassword`     | The password used to access the keystore        |
+| `spark.ssl.protocol`             |  Protocol (e.g. TLS)                            |
+| `spark.ssl.trustStore`           | must be trust.jks                               |
+| `spark.ssl.trustStorePassword`   | The password used to access the truststore      |
 
-Here are the required variables:
-
-| Variable                   | Description                                     |
-|----------------------------|-------------------------------------------------|
-| `spark.ssl.enabled`        | Whether to enable SSL (default: `false`).       |
-| `spark.ssl.keyStoreBase64` | Base64 encoded blob containing a Java keystore. |
 
 The Java keystore (and, optionally, truststore) are created using the [Java keytool][12]. The keystore must contain one private key and its signed public key. The truststore is optional and might contain a self-signed root-ca certificate that is explicitly trusted by Java.
 
-Both stores must be base64 encoded, for example:
+Both stores must be base64 encoded without newlines, for example:
 
-    cat keystore | base64 /u3+7QAAAAIAAAACAAAAAgA...
+```bash
+cat keystore | base64 -w 0 > keystore.base64
+cat keystore.base64
+/u3+7QAAAAIAAAACAAAAAgA...
+```
 
 **Note:** The base64 string of the keystore will probably be much longer than the snippet above, spanning 50 lines or so.
 
-With this and the password `secret` for the keystore and the private key, your JSON options file will look like this:
+Add the stores to your secrets in the DC/OS secret store. For example, if your base64-encoded keystores 
+and truststores are server.jks.base64 and trust.jks.base64, respectively, then use the following 
+commands to add them to the secret store: 
+
+```bash
+dcos security secrets create /__dcos_base64__truststore --value-file trust.jks.base64
+dcos security secrets create /__dcos_base64__keystore --value-file server.jks.base64
+```
 
-    {
-      "security": {
-        "ssl": {
-          "enabled": true,
-          "keyStoreBase64": "/u3+7QAAAAIAAAACAAAAAgA...”,
-          "keyStorePassword": "secret",
-          "keyPassword": "secret"
-        }
-      }
-    }
+In this case, you are adding two secrets `/truststore` and `/keystore`. 
+You must add the following configurations to your `dcos spark run ` command.
+The ones in parentheses are optional.:
```

Comment:
Good point, @keithchambers . Actually, that was the old verbose command. Here is the simplified one. What do you think? --